### DISCUSSION TITLE
Hit highlighting and gender conformity

### DISCUSSION THREAD

**Discussion by ferrarirosso** [microsoft-search/pnp-modern-search#2811] [General]
_This problem is probably located in SPO Search and not directly in PnP Search._

In my german speaking company, we have adopted the following pattern when writing textes gender conform, example for the word employee :
Mitarbeiter:in
If I create a Sharepoint Page with the following text :
Aber gendern im Text funktioniert leider nicht ich kann Mitarbeiter:in hier schreiben und die Doppelpunkte werden entfernt.

And I do a search for "ich kann", I get the following HitHighlightedSummary property (pnp search results wp/debug layout) :
`"HitHighlightedSummary": "<ddd/><c0>ich</c0> <c0>kann</c0> Mitarbeiterin hier schreiben und die Doppelpunkte werden entfernt. <ddd/><c0>ich</c0> <c0>kann</c0> Mitarbeiter:in hier schreiben und die Doppelpunkte werden entfernt.<ddd/><c0>ich</c0> <c0>kann</c0> Mitarbeiter:in hier<ddd/>",
`

1. Why is this hit returned 3 times ?
2. Why is the first hit changed and the colon removed ? (which is the problem I would like to solve)
3. Is there any way to circumvent this ?

Depending upon the searched pattern, the hit may also be returned only one time, and unfortunately "wrong" aka without the colon.

Thanks for your feedback !

**Comment by wobba** [comment]
This is on the API and I don't know of any good way to control this.

**Reply by ferrarirosso** [reply]
Thanks for your quick feedback !
